.env.example

```
@@ -55,13 +55,22 @@ RABBITMQ_HOST=192.168.250.31
 RABBITMQ_PORT=5672
 RABBITMQ_QUEUE_NAME=backtrade-queue
 
+# Queue retry config 
+QUEUE_RETRY_ENABLED=true
+QUEUE_RETRY_CRON="*/30 * * * * *"
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The QUEUE_RETRY_CRON key should go before the QUEUE_RETRY_ENABLED key

(UnorderedKey)